libr/util/sys.c

```diff
@@ -1175,6 +1179,20 @@ R_API char *r_sys_pid_to_path(int pid) {
 	if (ret != 0) {
 		return NULL;
 	}
+#elif __HAIKU__
+	char pathbuf[MAXPATHLEN] = {0};
```

Comment:
no need to initialize pathbuf. oh well, maybe just the first char.no the whole MAXPATHLEN